dockerfiles/postgres/fetch_schema.py

```diff
@@ -0,0 +1,49 @@
+import os
+import requests
+
```

Comment:
Use https://pypi.org/project/python-dotenv/ to fetch env values from Dockerfile

e.g

```python
from dotenv import load_dotenv

if __name__ == "__main__":
    load_dotenv()

    BB_URL = os.getenv("BB_URL")
    BB_SERVICE_ACCOUNT = os.getenv("BB_SERVICE_ACCOUNT")
    BB_SERVICE_KEY = os.getenv("BB_SERVICE_KEY")
    INSTANCE_ID = os.getenv("BB_INSTANCE_ID")
    DATABASE_ID = os.getenv("BB_DATABASE_ID")
```